doc/pipeline_details.rst

```diff
@@ -30,57 +30,57 @@ In the table below when an instruction stalls for X cycles X + 1 cycles pass bef
-|                       |                       | jump (which does the required flushing) so it has the same  |
-|                       |                       | stall characteristics (see above).                          |
-+-----------------------+-----------------------+-------------------------------------------------------------+
++-----------------------+--------------------------------------+-------------------------------------------------------------+
+|   Instruction Type    |             Stall Cycles             |                         Description                         |
++=======================+======================================+=============================================================+
+| Integer Computational | 0                                    | Integer Computational Instructions are defined in the       |
+|                       |                                      | RISCV-V RV32I Base Integer Instruction Set.                 |
++-----------------------+--------------------------------------+-------------------------------------------------------------+
+| CSR Access            | 0                                    | CSR Access Instruction are defined in 'Zicsr' of the        |
+|                       |                                      | RISC-V specification.                                       |
++-----------------------+--------------------------------------+-------------------------------------------------------------+
+| Load/Store            | 1 - N                                | Both loads and stores stall for at least one cycle to await |
+|                       |                                      | a response.  For loads this response is the load data       |
+|                       |                                      | (which is written directly to the register file the same    |
+|                       |                                      | cycle it is received).  For stores this is whether an error |
+|                       |                                      | was seen or not.  The longer the data side memory interface |
+|                       |                                      | takes to receive a response the longer loads and stores     |
+|                       |                                      | will stall.                                                 |
++-----------------------+--------------------------------------+-------------------------------------------------------------+
+| Multiplication        | 2/3 (Fast Multiplier)                | Fast: 2 for MUL, 3 for MULH.                                |
+|                       |                                      | Slow: clog2(``op_b``) for MUL, 32 for MULH                  |
+|                       | clog2(``op_b``)/32 (Slow Multiplier) | See details in :ref:`mult-div`                              |
```

Comment:
Can you please
- move the content in the middle column from Line 52 to Line 51, and
- add a full stop after MULH on Line 51?
The rendered documentation currently looks weird.

Reply:
OK will do. FYI the original also had the middle column text on Lines 50 and 52 with 51 empty, not sure how that affects rendering of the doc exactly.

Reply:
It looks like this:

![Screenshot from 2020-02-10 13-00-29](https://user-images.githubusercontent.com/20307557/74148267-5bfbd680-4c05-11ea-9417-586098b19fab.png)

Reply:
Hmm yes, the whole clog2 thing is a bit long, you could trade off accuracy for legibility and use "up to 32" for the slow multiplier part. Not sure whether that's desired though.

Reply:
I've filed https://github.com/lowRISC/ibex/issues/595 to make the table easier to edit in the future. For now, do whatever is easiest to get the right rendering, and don't worry to much about it looking "ugly" in source code.